disco_aws_automation/disco_elb.py

```diff
@@ -367,7 +367,7 @@ def _describe_instance_health(self, elb_id, instance_ids=None):
         return instances["InstanceStates"]
 
     def wait_for_instance_health_state(self, hostclass, testing=False, instance_ids=None, state="InService",
-                                       timeout=180):
+                                       timeout=600):
```

Comment:
This seems like a temporary fix till we find the root cause for the timeout, why instances taking more than 3 mins to come up.